homeassistant/components/planifneige/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""Platform for the City of Montreal's Planif-Neige snow removal APIs."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.const import CONF_API_KEY, CONF_NAME, CONF_SCAN_INTERVAL
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+
+DOMAIN = 'planifneige'
+DATA_PLANIFNEIGE = 'data_planifneige'
+
+DATA_UPDATED = '{}_data_updated'.format(DOMAIN)
+
+PLANIFNEIGE_ATTRIBUTION = "Information provided by the City of Montreal "
+
+REQUIREMENTS = ['planif-neige-client==0.1.1']
```

Comment:
Please move this to the new `manifest.json` https://developers.home-assistant.io/docs/en/creating_integration_manifest.html